chapter2/tests/t20/terraform/dev/alb.tf

```diff
@@ -0,0 +1,45 @@
+module "alb" {
+  source             = "terraform-aws-modules/alb/aws"
+  version            = "~> 8.6.1"
+  name               = local.labels.alb
+  load_balancer_type = "application"
+  vpc_id             = data.aws_vpc.target.id
+  subnets            = data.aws_subnets.wordpress.ids
+  security_groups    = [module.wordpress_alb_sg.security_group_id]
+  tags               = var.tags
+
+  target_groups = [
+    {
```

Comment:
taregt_groups and http_listeners should be similar in attr sequence

listeners:
- port
- protocol
- others

target_groups should be similar
- backend_protocol
- backend_port
- name
- others

Reply:
> 

@dmitry-mightydevops
Done